clients/imodels-client-management/src/base/types/apiEntities/UserInterfaces.ts

```diff
@@ -20,6 +20,18 @@ export interface MinimalUser {
   _links: MinimalUserLinks;
 }
 
+/** User statistics */
+export interface UserStatistics {
+  /** Number of changesets pushed by the user. */
```

Comment:
we use uppercase in entity names (Changeset, Briefcase)

Reply:
updated.